CHANGELOG.md

```diff
@@ -1,13 +1,23 @@
 # Changelog
 
-## [Unreleased]
+## [0.20.0]
+
+### Added
```

Comment:
Should we also mention the UPD support from dumbo? (https://github.com/firecracker-microvm/firecracker/commit/03e61c5d7088aaa4612e6bfa984490f75acfb3a2)

Reply:
I don't think so. Doesn't seem customer facing.